### DISCUSSION TITLE
how to integrate with abp vnext ?

### DISCUSSION THREAD

**Discussion by wjkhappy14** [elsa-workflows/elsa-core#1278] [Q&A]
can you provider a module for abp vnext ? [](https://github.com/abpframework/abp)

**Comment by wjkhappy14** [comment]
https://github.com/abpframework/abp

**Comment by sfmskywalker** [comment]
Why not. We are [also doing this](https://github.com/elsa-workflows/elsa-orchard-core) for [Orchard Core](https://www.orchardcore.net/). It would make sense to have a module for ABP too. I also want to add a module for [Oqtane](https://www.oqtane.org/).

**Comment by wjkhappy14** [comment]
so great job！
Erwartungsvoll gespannt！

**Comment by jortygu** [comment]
> Why not. We are [also doing this](https://github.com/elsa-workflows/elsa-orchard-core) for [Orchard Core](https://www.orchardcore.net/). It would make sense to have a module for ABP too. I also want to add a module for [Oqtane](https://www.oqtane.org/).

great job! Any Plan or Schedule?

**Comment by sfmskywalker** [comment]
Nothing concrete at the moment. My top priority is reworking the V2 designer + V2 Dashboard. After that, it depends on various factors. An important factor being what is the most asked for. In any case, I wouldn't expect this to be available until at least Q3. Of course, anyone is welcome to submit a PR to speed things up a little.

**Comment by gdlcf88** [comment]
See https://github.com/EasyAbp/Elsa
